sample: tf-m psa template redefine purpose (#9522)
nrfconnect/sdk-nrf: samples/tfm/tfm_psa_template/README.rst

.. _tfm_psa_template:

TF-M: PSA Template
##################

.. contents::
   :local:
   :depth: 2

> This sample gives a template on how to follow best practices according to the Arm PSA on nRF devices.

Reviewer: ```suggestion
This sample provides a template for Arm Platform Security Architecture (PSA) best practices on nRF devices.
```

.. _tfm_psa_template:

> TF-M: PSA Template

Reviewer: ```suggestion
TF-M: PSA template
```

> ###########################

Reviewer 2: Not an RST expert, but I think the ### needs to be the same length as the title.

Overview
********

> This sample uses Trusted Firmware-M, nRF Secure Immutable bootloader and MCUboot to demonstrate how to implement the best practices given in the Arm Platform Security Architecture (PSA) requirements.

Reviewer: ```suggestion
This sample uses Trusted Firmware-M, nRF Secure Immutable bootloader and MCUboot to demonstrate how to implement the best practices that comply with the Arm PSA requirements.
```

Overview
********

> This sample uses Trusted Firmware-M, nRF Secure Immutable bootloader and MCUboot to demonstrate on how to implement the best practices given in the Arm Platform Security Architecture (PSA) requirements.

Reviewer 3: ```suggestion
This sample uses Trusted Firmware-M, nRF Secure Immutable bootloader and MCUboot to demonstrate how to implement the best practices given in the Arm Platform Security Architecture (PSA) requirements.
```

> It includes to provision the device with keys and been able to perform an device firmware update.

Reviewer 4: ```suggestion
It includes provisioning the device with keys and being able to perform a device firmware update.
```

    west build -b nrf5340dk_nrf5340_cpuapp nrf/samples/tfm/provisioning_image -d build_provisioning_image
    west flash --erase -d build_provisioning_image

> Build and flash the TF-M PSA Template sample.

Reviewer: ```suggestion
Build and flash the TF-M PSA template sample.
```
